spec/support/sass_support.rb

```diff
@@ -0,0 +1,10 @@
+module SassSupport
+  def generate_css
+    _mkdir('tmp')
+    `sass -I . --update spec/fixtures:tmp`
+  end
+
+  def clean_up
+    FileUtils.rm_rf('tmp')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.